capnslog/formatters.go

```diff
@@ -28,6 +29,28 @@ type Formatter interface {
 	Flush()
 }
 
+func NewSimpleFormatter(w io.Writer) *SimpleFormatter {
+	return &SimpleFormatter{
+		w: bufio.NewWriter(w),
```

Comment:
This I know, but either I misunderstand where the git comments are pointing to, or I don't see the lack of capitalization... Are we talking about: NewSimpleFormatter ? It looks capitalized to me.